connectors/slack.rb

```diff
@@ -35,13 +35,18 @@ def update_users(users)
       if Linkbot.user_exists?(user_id)
         Linkbot.update_user(user["name"],user_id)
       else
+        puts "Adding #{user["name"]}"
         Linkbot.add_user(user["name"],user_id)
       end
     end
   end
 
   def process_message(time, nick, text, options={})
-    if Linkbot.user_exists?(nick)
+    if (!nick.nil? && nick != "")
```

Comment:
This is the sort of thing I liked ActiveSupport's [`Object.present?`](https://github.com/rails/rails/blob/v4.2.1/activesupport/lib/active_support/core_ext/object/blank.rb#L23-L25).

< _Robb hides_ >